Header header
# Array of target positions
geometry_msgs/Point[] targets
